File: SRP/Python/Journal/WithoutViolation/journal.py
class Journal:
    def __init__(self) -> None:
        self.entries: list(str) = []
        self.count: int = 0

    def add_entry(self, text: str) -> None:
        self.count += 1
        self.entries.append(f"{self.count}: {text}")

    def remove_entry(self, pos: int) -> None:
        del self.entries[pos]

    def __str__(self) -> str:
        return "\n".join(self.entries)

File: SRP/JavaScript/WithoutViolation/journal.js
class Journal {
  #count = 0;

  constructor() {
    this.entries = [];
  }

  addEntry(text) {
    let count = ++this.#count;
    let entry = `${count}: ${text}`;
    this.entries[count] = entry;
    return count;
  }

  removeEntry(index) {
    delete this.entries[index];
  }

  toString() {
    return Object.values(this.entries).join("\n");
  }
}

module.exports.Journal = Journal;
